README.md

```diff
@@ -139,28 +157,54 @@ After that the most recent value will always be stored in the beginning of parti
 SELECT sensor_id, temperature, registered_at, room_name
 FROM test.test
 WHERE sensor_id IN (99051fe9-6a9c-46c2-b949-38ef78858dd0, 99051fe9-6a9c-46c2-b949-38ef78858dd0)
-AND registered_at > $__timeFrom and registered_at < $__timeTo
-ORDER BY registered_at
+AND registered_at > $__timeFrom AND registered_at < $__timeTo
 PER PARTITION LIMIT 1
 ```
 Note that `PER PARTITION LIMIT 1` used instead of `LIMIT 1` to query one row for each partition and not just one row total.
 
+### Annotations
+[Grafana Annotations documentation](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/annotate-visualizations/)
+
+### Alerting
+Alerting is supported, however it has some limitations. Grafana does not support long(narrow) series in alerting, 
+so query result must be converted to wide series before hading it over to grafana. Datasource performs it in pretty
+simple way - it creates labels using all the non-timeseries field and then removes that fields from response.
+Basically, this query(using example table)
+```
+SELECT sensor_id, temperature, registered_at, location
+FROM test.test
+WHERE sensor_id IN (99051fe9-6a9c-46c2-b949-38ef78858dd0, 99051fe9-6a9c-46c2-b949-38ef78858dd0)
+AND registered_at > $__timeFrom AND registered_at < $__timeTo
```

Comment:
Maybe we should mention that user **MUST** use timeFrom/timeTo filters to avoid excessive data fetching? Now it looks more like "could".

Reply:
I believe it is still an optional parameter. It will look like **every** query **must** have time filter which is not true. But we could put some advice on that matter in `Tips and tricks` section. I'll think about it.

Reply:
In my VariableEditor it looks like that:

<img width="720" alt="image" src="https://github.com/HadesArchitect/GrafanaCassandraDatasource/assets/1742301/ad421ca3-59f3-4980-bc31-bf8823e4de04">

Reply:
It just works!:) Grafana has built-in variables `$__from`/`$__to` that are interpolated by grafana itself. And thay also have neat formatting feature: https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#__from-and-__to
So, query like that
```
SELECT sensor_id, temperature, registered_at, location 
FROM test.test 
WHERE sensor_id IN (99051fe9-6a9c-46c2-b949-38ef78858dd1, 99051fe9-6a9c-46c2-b949-38ef78858dd0)
AND registered_at > '${__from:date:YYYY-MM-DD}' 
AND registered_at < '${__to:date:YYYY-MM-DD}' + 1d
```
turns into
```
SELECT sensor_id, temperature, registered_at, location
FROM test.test
WHERE sensor_id IN (99051fe9-6a9c-46c2-b949-38ef78858dd1, 99051fe9-6a9c-46c2-b949-38ef78858dd0)
AND registered_at > '2023-10-17'
AND registered_at < '2023-10-17' + 1d
```

Totally automagically, nothing else to do!